DittoHealthMetrics/build.gradle

```diff
@@ -11,15 +11,27 @@ apply from: "$rootProject.projectDir/gradle/deploy.gradle"
 apply from: "$rootProject.projectDir/gradle/android-common.gradle"
 
 android {
-    namespace 'live.ditto.dittohealthmetrics'
+    namespace "live.ditto.health"
 }
 
 dependencies {
+    androidTestImplementation libs.androidx.test.ext.junit
+
+    implementation libs.androidx.compose.material3.material3
+    implementation libs.androidx.compose.ui.ui
+    implementation libs.androidx.compose.ui.uiTooling
+    implementation libs.androidx.compose.ui.uiToolingPreview
+    implementation libs.androidx.navigation.navigationCompose
+    implementation libs.live.ditto.ditto
+    // Dependency constraint for ditto
+    constraints {
+        implementation(libs.live.ditto.ditto) {
+            version {
+                strictly "[4.5.0,)"
```

Comment:
Should this also bump to 4.7.0+?

Reply:
Yes good catch!

Reply:
Question actually - do we want ALL tools to use a min. version of 4.7.0+? Or just the ones that use the peerKeyString?

Reply:
Hmm, good question - on the Swift side, it's all or nothing because the tools are all bundled as a single SPM package, so I didn't consider that.

If we can do these independently and really don't need anything from 4.7 for some tools, let's do it to make as many tools as possible as usable as possible.

Reply:
Okay, I've updated the min version for the specific tools that use the peerkey string.